libs/log/tmfmt_logger.go

```diff
@@ -98,13 +98,13 @@ func (l tmfmtLogger) Log(keyvals ...interface{}) error {
 	// Form a custom CometBFT line
 	//
 	// Example:
-	//     D[2016-05-02|11:06:44.322]   Stopping AddrBook (ignoring: already stopped)
+	//     DEBUG[2016-05-02|11:06:44.322]   Stopping AddrBook (ignoring: already stopped)
 	//
 	// Description:
-	//     D										- first character of the level, uppercase (ASCII only)
+	//     DEBUG						- log level, uppercase
 	//     [2016-05-02|11:06:44.322]    - our time format (see https://golang.org/src/time/format.go)
 	//     Stopping ...					- message
-	enc.buf.WriteString(fmt.Sprintf("%c[%s] %-44s ", lvl[0]-32, time.Now().Format("2006-01-02|15:04:05.000"), msg))
+	enc.buf.WriteString(fmt.Sprintf("%-5s[%s] %-44s ", strings.ToUpper(lvl), time.Now().Format("2006-01-02|15:04:05.000"), msg))
```

Comment:
We actually did it to bring the logging in unison with bor. See this [PR](https://github.com/maticnetwork/tendermint/pull/25). 
I guess we could live without it but I'll confirm once. But out of curiosity, any reason why comet (and tendermint previously) didn't go with printing the whole log level ?

Reply:
Not sure. I would bet on conciseness and on all the "headers" being aligned independent of the log levels, instead of varying for example if using INFO and ERROR.

Please keep this change for now. I've added an issue proposing upstreaming a configuration parameter to specify the format (https://github.com/cometbft/cometbft/issues/2012)

Reply:
Pls feel free to mark as resolved